docs/general/wallets.md

```diff
@@ -7,40 +7,43 @@ keywords: [wallets, account, signer, supported wallets, balance, token]
 slug: ../wallets
 ---
 
-Navigating the space of wallets can be overwhelming, if you are new to blockchain technology. In
-general, an account on a typical blockchain network is a public-private key pair. Access to a
-private key gives full access to all the allowed transactions on that blockchain account. It is
-important to keep the private key secure.
-
-:::tip Not your keys, not your tokens!
+:::caution
 
-With custodial wallets (like accounts central exchanges), another party controls your private keys.
-With non-custodial wallets, only you have access to your account's private key.
+Using your due diligence in researching and using the wallets listed below would be best. For any
+issues related to these wallet, reach out to their support teams directly.
 
 :::
 
-To realize the multichain vision of Polkadot, it is important to have non-custodial wallets that
-make it convenient to interact with multiple blockchains in Polkadot ecosystem.
+If you are new to blockchain technology, generally a typical blockchain network account is a
+public-private key pair. Access to a private key gives full access to all the allowed transactions
+on that blockchain account. It is essential to keep the private key secure.
 
-## Non-Custodial Wallets
+Typically, the account keys are either stored and accessed through a browser extension or a
+smartphone app (which are considered as a hot wallets as they are online), or an air-gapped device
+or a hardware wallet (which are considered as cold wallets as they are offline).
 
-:::caution
+:::tip Not your keys, not your coins!
```

Comment:
Still prefer tokens instead of coins here. None of the tokens are referred to as coins in Polkadot ecosystem.